infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/endpoints/EndpointMetadata.java

```diff
@@ -156,6 +158,29 @@ public <T> T getRequestBody(final InputStream body, final Optional<String> maybe
     }
   }
 
+  @SuppressWarnings({"unchecked", "TypeParameterUnusedInFormals"})
+  public <T> T getRequestBody(final InputStream body, final Map<String, String> headers)
```

Comment:
We may want to deprecate the other `getRequestBody` method to avoid using it in tests from now on.